LayoutTests/fast/css/view-transitions-nested-transparency-layers-expected.html

```diff
@@ -0,0 +1,19 @@
+<style>
+    .parent {
+        width: 100px;
+        height: 100px;
+        background: red;
+        opacity: 0.5;
```

Comment:
```suggestion
```

The snapshot hopefully doesn't include the opacity from the ancestor, so we need to remove it here too. Removing the background just makes the test simpler.